packages/react-components/react-dialog/src/stories/DialogChangeFocus.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Dialog, DialogTrigger, DialogSurface, DialogTitle, DialogBody, DialogActions } from '@fluentui/react-dialog';
+import { Button } from '@fluentui/react-components';
+
+export const ChangeFocus = () => {
```

Comment:
```suggestion
export const CustomFocusedElementOnOpen = () => {
```

nit: is this a better name ?